code/game/machinery/recharger.dm

```diff
@@ -10,7 +10,7 @@ obj/structure/machinery/recharger
 	active_power_usage = 15000	//15 kW
 	var/obj/item/charging = null
 	var/percent_charge_complete = 0
-	var/list/allowed_devices = list(/obj/item/weapon/melee/baton, /obj/item/cell, /obj/item/weapon/gun/energy/taser, /obj/item/device/defibrillator)
+	var/list/allowed_devices = list(/obj/item/weapon/melee/baton, /obj/item/cell, /obj/item/weapon/gun/energy, /obj/item/device/defibrillator)
```

Comment:
There's still checks further down in process() of recharger to taser's. So my understanding is that this makes you able to put other energy guns in (including pred rifle/casters), but they don't actually charge

Reply:
https://github.com/cmss13-devs/cmss13/blob/d51c243cdb55cda3068f318244234196452be797/code/game/machinery/recharger.dm#L76